.github/workflows/db-prod-ci-cd.yaml

```diff
@@ -0,0 +1,21 @@
+name: "Database CICD Prod"
```

Comment:
This needs to be combined with the proj-ci-cd else the pipeline will not be sequential

Reply:
I agree with this